demos/python_demos/asl_recognition_demo/asl_recognition_demo/common.py

```diff
@@ -45,19 +45,33 @@ def __init__(self, model_path, device, ie_core, num_requests, output_name=None):
             raise RuntimeError("Following layers are not supported by the {} plugin:\n {}"
                                .format(device, ', '.join(not_supported_layers)))
 
+        self.exec_net = ie_core.load_network(network=self.net,
+                                             device_name=device,
+                                             num_requests=num_requests)
+
         self.input_name = next(iter(self.net.inputs))
-        if len(self.net.outputs) > 1 and output_name is not None:
-            if output_name is not None:
-                self.output_name = output_name
+        if len(self.net.outputs) > 1:
+            if output_shape is not None:
+                candidates = []
+                for candidate_name in self.net.outputs:
+                    candidate_shape = self.exec_net.requests[0].outputs[candidate_name].shape
+                    if len(candidate_shape) != len(output_shape):
+                        continue
+
+                    matches = [src == trg if trg is not None and trg > 0 else True
```

Comment:
```suggestion
                    matches = [src == trg or trg < 0
```

It doesn't look like `trg` can be `None`.

Reply:
Fixed.